.github/workflows/update_types.yml

```diff
@@ -0,0 +1,75 @@
+name: Check Saleor release and update types
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 18 * * 1-5" # workdays at 18:00
+
+jobs:
+  check_saleor_release:
+    name: Check Saleor release and update types
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
```

Comment:
Can we move this step after checking if there's a new release? Should be faster

Reply:
Done